migrations/versions/ff3141aecdd4_.py

```diff
@@ -0,0 +1,37 @@
+"""Update Olink's assay_raw_ct artifact data format from XLSX to CSV.
+
+Revision ID: ff3141aecdd4
+Revises: b8eaf567ac2f
+Create Date: 2019-11-14 11:43:49.297901
+
+"""
+import os
+
+from cidc_schemas.migrations import v0_10_0_to_v0_10_2
+from cidc_api.utils.migrations import run_metadata_migration
+
+# revision identifiers, used by Alembic.
+revision = "ff3141aecdd4"
+down_revision = "b8eaf567ac2f"
+branch_labels = None
+depends_on = None
+
+is_testing = os.environ.get("TESTING")
+
+
+def upgrade():
+    """Update Olink's assay_raw_ct artifact data format to CSV"""
+    # Don't run this migration on the test database
+    if is_testing:
```

Comment:
not sure why it shouldn't. because it's supposed to be empty?

Reply:
It _will_ be empty, but the reason why is that this migration accesses GCS resources that Travis does not have access to, so running it in CI would fail.

That being said, this detail should probably be encapsulated inside `run_metadata_migration`.